CHANGES

```diff
@@ -24,6 +24,7 @@
   attributes which give a citation for astroquery in bibtex format. [#1391]
 - VIZIER: Support using the output values of ``find_catalog`` in
   ``get_catalog``. [#603]
+- Vizier: Add 'datalink' column (link to FITS) when present (#1455)
```

Comment:
not sure what's the changelog bot's issue, I suspect it may be not as forgiving for the parens vs bracket usage as the release script...